object-construction-checker/tests/mustcall/CommonModuleCrash.java

```diff
@@ -1,11 +1,9 @@
 import java.net.*;
 
 class CommonModuleCrash {
-    // :: error: required.method.not.called
```

Comment:
why this change?

Reply:
Because new Socket creates unbound socket which is MCC{}